share/ansible_navigator/utils/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Utility scripts that will be mounted into the execution environment.
```

Comment:
I don't think we need future possibilities in the title. Just tell people what it does not what something external does with it in some cases. At least, not in the title.
```suggestion
"""Utility scripts mounted into the execution environment.
```